### PR TITLE
remove `securityContext` from sample windows deployment

### PR DESCRIPTION
**Pull request by tosmi**
in a 4.11 cluster with WMCO 6.0.0 on AWS (worker node with Windows 2019) the deployment fails with the following message:

```
Error: container's runAsUser () which will be regarded as root identity and will break non-root policy (pod: "win-webserver-5c9bb5c774-ng7qn_windows-workload(9cc9f0be-9146-405b-b0da-c871e8a3fe43)", container: windowswebserver)
```

AFAIK privileged containers are currently not supported (see https://docs.openshift.com/container-platform/4.11/windows_containers/windows-containers-release-notes-6-x.html#windows-containers-release-notes-limitations_windows-containers-release-notes) and for me the deployment fails anyways.

Version(s):

Issue:

Link to docs preview:

Additional information:

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@mburke5678, will you please run this by QE and help get it merged if QE approves it?

**Review by jfrancoa — APPROVED** [review]
/lgtm

**Comment by mburke5678** [comment]
@aravindhp Can you PTAL to help clarify which OCP versions this change applies to?

**Comment by aravindhp** [comment]
>AFAIK privileged containers are currently not supported

The security context here just makes the user to be `ContainerAdministator`, It does not make the Pod privileged. From what I remember if you don't use this user, the webserver will not come up. We should look into this a bit before we decided to remove this from the example as we don't see this issue in our e2e tests.

**Comment by mburke5678** [comment]
@jfrancoa Got the same error in his testing earlier today.

**Comment by jfrancoa** [comment]
> @jfrancoa Got the same error in his testing earlier today.

Right, I did a quick test with a very similar deployment config and got the very same error:
```
[jfrancoa@localhost wmco]$ oc get pods
NAME                              READY   STATUS                       RESTARTS   AGE
linux-webserver-8fc5c86b8-pmld6   1/1     Running                      0          150m
win-webserver-585d794b5-ctszh     0/1     CreateContainerConfigError   0          6s
win-webserver-585d794b5-grwns     0/1     CreateContainerConfigError   0          6s
win-webserver-585d794b5-m98fm     0/1     CreateContainerConfigError   0          6s
win-webserver-585d794b5-v4sqd     0/1     CreateContainerConfigError   0          6s
win-webserver-585d794b5-vk5ff     0/1     CreateContainerConfigError   0          6s
  

Warning  Failed     3s (x3 over 15s)  kubelet            Error: container's runAsUserName (ContainerAdministrator) which will be regarded as root identity and will break non-root policy (pod: "win-webserver-585d794b5-ctszh_winc-test(191f42f3-f1cd-4b03-ae79-18e111ae4d4b)", container: windowswebserver)
```

```
---
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: win-webserver
  name: win-webserver
spec:
  selector:
    matchLabels:
      app: win-webserver
  replicas: 5
  template:
    metadata:
      labels:
        app: win-webserver
      name: win-webserver
    spec:
      tolerations:
      - key: "os"
        value: "Windows"
        Effect: "NoSchedule"
      containers:
      - name: windowswebserver
        image: mcr.microsoft.com/windows/servercore:ltsc2019
        imagePullPolicy: IfNotPresent
        securityContext:
          windowsOptions:
            runAsUserName: "ContainerAdministrator"
        command:
        - powershell.exe
        - -command
        - $listener = New-Object System.Net.HttpListener; $listener.Prefixes.Add('http://*:80/'); $listener.Start();Write-Host('Listening at http://*:80/'); while ($listener.IsListening) { $context = $listener.GetContext(); $response = $context.Response; $content='<html><body><H1>Windows Container Web Server</H1></body></html>'; $buffer = [System.Text.Encoding]::UTF8.GetBytes($content); $response.ContentLength64 = $buffer.Length; $response.OutputStream.Write($buffer, 0, $buffer.Length); $response.Close(); };
      nodeSelector:
        beta.kubernetes.io/os: windows

```

File: modules/sample-windows-workload-deployment.adoc [hunk 1]
```diff
@@ -63,9 +63,6 @@ spec:
         - powershell.exe
         - -command
         - $listener = New-Object System.Net.HttpListener; $listener.Prefixes.Add('http://*:80/'); $listener.Start();Write-Host('Listening at http://*:80/'); while ($listener.IsListening) { $context = $listener.GetContext(); $response = $context.Response; $content='<html><body><H1>Red Hat OpenShift + Windows Container Workloads</H1></body></html>'; $buffer = [System.Text.Encoding]::UTF8.GetBytes($content); $response.ContentLength64 = $buffer.Length; $response.OutputStream.Write($buffer, 0, $buffer.Length); $response.Close(); };
-        securityContext:
-          windowsOptions:
```
[the remark this answers is not in the document]
Comment (tosmi) [reply modules/sample-windows-workload-deployment.adoc:67]: ah, sorry i forgot about this, i think i've also changed the port to 8080 in the example while tesing. afair, this fixed the "access denied" issue.

[the remark this answers is not in the document]
Comment (mburke5678) [reply modules/sample-windows-workload-deployment.adoc:67]: I will make the change in https://github.com/openshift/openshift-docs/pull/51376

**Pull request closed without merging**